src/core/killall.c

```diff
@@ -177,8 +178,11 @@ static int killall(int sig, Set *pids, bool send_sighup) {
                 }
 
                 if (kill(pid, sig) >= 0) {
-                        if (pids)
-                                set_put(pids, ULONG_TO_PTR(pid));
+                        if (pids) {
+                                r = set_put(pids, ULONG_TO_PTR(pid));
+                                if (r < 0)
+                                        log_oom();
```

Comment:
The `(void)` is just to tell the compiler we're not interested in the `-ENOMEM` usually returned by `log_oom()`.

Under `-Wunused-result` this statement (without the `(void)`) might generate a compiler warning...

Otherwise, it looks good, to me.